.github/workflows/build.yml

```diff
@@ -16,18 +16,17 @@ jobs:
     strategy:
       matrix:
         os:
-          - ubuntu-16.04 # https://launchpad.net/~hvr/+archive/ubuntu/ghc?field.series_filter=xenial
+          - ubuntu-latest
```

Comment:
CI was failing because no build boxes were available for this for over a day. Trying this to see if it'll work